proposals/0000-reduce-fragmentation.md

```diff
@@ -190,151 +339,448 @@ const autoCompleteIOS = {
-  * Support subtree-level logical properties, where setting the `dir` prop to `ltr` or `rtl` alters the way logical properties are resolved. Currently, React Native only supports global writing direction configuration for Android.
-* CSS Animations
-  * Support declarative keyframes and animations that can be optimized on the native side and avoid the need for `Animated`.
-  * See [animation](https://developer.mozilla.org/en-US/docs/Web/CSS/animation)).
-* CSS Transitions
-  * Support declarative transitions that can be optimized on the native side and avoid the need for `Animated`.
-  * Support `transitionProperty`, `transitionDuration`, `transitionTimingFunction`, and `transitionDelay` (see [transition](https://developer.mozilla.org/en-US/docs/Web/CSS/transition)).
-* `aspectRatio`
-  * Support string values, i.e., `'16 / 9'`, to align with CSS.
-* `backgroundImage`
-  * Support setting background images via `url()`.
-  * Optional: support CSS gradients.
-* `boxShadow`
-  * Add native support for CSS box shadows.
-  * This would replace the very buggy iOS-specific `shadow*` styles and Android `elevation` style, allowing developers to unify shadows across native and web platforms.
-* `filter`
- * Support declarative filters per [filter](https://developer.mozilla.org/en-US/docs/Web/CSS/filter)
-* `fontVariant`
-  * Support space-separated string values to align with CSS.
-* `fontWeight`
-  * Support number values to align with React DOM / CSS.
-* `lineClamp`
-  * Support CSS clamping text to a set number of lines via style.
-  * This would be equivalent to using the `numberOfLines` prop on `<Text>` components.
-* `objectFit`
-  * Support CSS `objectFit` property for `<Image>`.
-  * This would be a subset (no "repeat" keyword) of functionality currently supported by the `resizeMode` prop and style for `<Image>`.
-  * `objectFit === 'contain'` => `resizeMode = 'contain'`
-  * `objectFit === 'cover'` => `resizeMode = 'cover'`
-  * `objectFit === 'fill'` => `resizeMode = 'stretch'`
-  * `objectFit === 'scale-down'` => `resizeMode = 'contain'`
-* `pointerEvents`
-  * Support setting the `pointerEvents` values via style rather than a prop.
-  * Retain the React Native specific `box-none` and `box-only` values.
-* `textShadow`
-  * Add native support for CSS text shadows.
-* `transform`
-  * Support using string values to set transforms.
-  * This can be shimmed in user-space with a parser.
-* `verticalAlign`
-  * Make `verticalAlign` the same as `textAlignVertical`.
-  * `verticalAlign === 'middle'` is the same as `textAlignVertical = 'center'`;
-* `userSelect`
-  * Support setting ability to select text via style.
-  * Equivalent to using `selectable` prop on `<Text>`.
+-----
+
+# Styles API (CSS subset)
+
+## StyleSheet
+
+* [ ] Deprecate `StyleSheet.absoluteFill`. Use `position:'absolute';inset:0;` instead.
+* [ ] Deprecate `StyleSheet.absoluteFillObject`.
+* [ ] Deprecate `StyleSheet.compose()`. Use existing `[ a, b ]` syntax instead.
```

Comment:
Compose is useful for internal micro optimizations and avoid array allocations. Maybe we could make it an internal-only API, which is stubbed / ignored for web?

Reply:
How much of a difference do those micro optimisations make it product code rarely uses it? Not to mention the API is limited to only 2 args